l10n_nl_tax_statement_icp/views/l10n_nl_vat_statement_view.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2018 Onestein (<https://www.onestein.eu>)
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_l10n_nl_vat_report_form" model="ir.ui.view">
+        <field name="model">l10n.nl.vat.statement</field>
+        <field
+            name="inherit_id"
+            ref="l10n_nl_tax_statement.view_l10n_nl_vat_report_form"
+        />
+        <field name="arch" type="xml">
+            <xpath expr="//page[@name='statement']">
+                <group
+                    attrs="{'invisible': ['|',('state','!=','draft'),('parent_id','=',False)]}"
+                >
+                    <div
+                        states="draft"
+                    >Post the statement to create the ICP report!</div>
+                    <button
+                        name="post"
+                        string="Post"
+                        states="draft"
+                        type="object"
+                        class="oe_stat_button"
+                        icon="fa-arrow-right text-success"
+                    />
+                </group>
+            </xpath>
+            <xpath expr="//notebook" position="inside">
+                <page
+                    name="icp_statement"
+                    string="ICP Statement"
+                    attrs="{'invisible':[('state','=','draft')]}"
```

Comment:
I think it should always be visible. It can be confusing if the ICP is not yet shown.
The ICP lines also act as a check to compare to the statement. The user can press the "update" button whenever needed.

Reply:
Yes I agree. This actual solution was made to keep code simpler. I will work on your proposal when I will find some time.

Reply:
@casvissers done, would you give it a try?